plugin-server/src/utils/db/db.ts

```diff
@@ -1312,11 +1327,8 @@ export class DB {
                 `
                 SELECT * FROM posthog_eventdefinition
                 ${teamId ? 'WHERE team_id = $1' : ''}
-                -- Order by something that gives a deterministic order. Note
-                -- that this is a unique index.
-                ORDER BY (team_id, name)
                 `,
-                teamId ? [teamId] : undefined,
+                [teamId],
```

Comment:
These changes were needed as otherwise the tests broke, iirc. Unless we have also changed the tests that failed due to this?